core/src/main/resources/hudson/model/Messages_pl.properties

```diff
@@ -33,7 +33,7 @@ Job.BuildStability=Stabilność zadań: {0}
 Job.NOfMFailed={0} spośród {1} ostatnich zadań nie powiodło się.
 Job.NoRecentBuildFailed=Brak ostatnich zadań zakończonych niepowodzeniem.
 AbstractProject.Pronoun=Projekt
-AbstractItem.Pronoun=Projekt
+AbstractItem.Pronoun=projekt
```

Comment:
Hi,
I'm wondering why did you changed this to lowercase? 
Also this class refers to all kind of items, so maybe instead of `project` we should use something closer to `item` like `obiekt` or `element`? 
What do you think?

Reply:
There are two items here: line 35 with upper case and 36 with lower.
35 -> `Project` myFooProject
36 -> Delete `project`

and what I have observed there is no need to introduce new translation for `project` but that would be good idea for `view` or `build`. IMO having placeholder like `Delete {}` that is used for any object/item does not help to make good localizations :)

Reply:
sad.